### ISSUE TITLE
emmet.vim does not work on my gVim

### ISSUE THREAD

**Issue by liushuping** [mattn/emmet-vim#200]
I have emmet.vim installed on my gVim (v7.2 on Windows 7) by following installation guide. However, I tried and it does not work, did I have something miss configured? please help

Below is the configuration in _gvimrc file
filetype off
set rtp+=$VIM/vimfiles/bundle/vundle/ 
call vundle#rc('$VIM/vimfiles/bundle/') 
Bundle 'gmarik/vundle' 
Bundle "mattn/emmet-vim"
Bundle "jelera/vim-javascript-syntax"
Bundle "pangloss/vim-javascript"
Bundle "Raimondi/delimitMate"
Bundle "Valloric/YouCompleteMe"
filetype plugin indent on 

An here is the folder structure
C:\Vim
C:\Vim\vim72
C:\Vim\vimfiles
C:\Vim\vimfiles\bundle
C:\Vim\vimfiles\bundle\emmet-vim
C:\Vim\vimfiles\bundle\emmet-vim\autoload
C:\Vim\vimfiles\bundle\emmet-vim\doc
C:\Vim\vimfiles\bundle\emmet-vim\plugin
C:\Vim\vimfiles\bundle\emmet-vim\emmet.vim.vimup
C:\Vim\vimfiles\bundle\emmet-vim\Makefile
C:\Vim\vimfiles\bundle\emmet-vim\README.mkd
C:\Vim\vimfiles\bundle\emmet-vim\TODO
C:\Vim\vimfiles\bundle\emmet-vim\TUTORIAL
C:\Vim\vimfiles\bundle\emmet-vim\unittest.vim

**Comment by izoj2013** [comment]
I know this post is quite old; I stumbled on emmet-vim because I have trying some web prototypes.
I installed emmet-vim using Vundle and tried the first tutorial (copied below):

Type ("_" is the cursor position):

html:5_

Then type <c-y>, (Ctrly,), and you should see:

It is not working.

Here is part of ~/.vim/bundle directory:

|-- autoload
|   |-- emmet
|   |   |-- lang
|   |   |   |-- css.vim
|   |   |   |-- haml.vim
|   |   |   |-- html.vim
|   |   |   |-- jade.vim
|   |   |   |-- less.vim
|   |   |   |-- sass.vim
|   |   |   |-- scss.vim
|   |   |   `-- slim.vim
|   |   |-- lang.vim
|   |   |-- lorem
|   |   |   |-- en.vim
|   |   |   `-- ja.vim
|   |   `-- util.vim
|   `-- emmet.vim
|-- doc
|   |-- emmet.txt
|   `-- tags
|-- emmet.vim.vimup
|-- Makefile
|-- plugin
|   `-- emmet.vim
|-- README.mkd
|-- TODO
|-- TUTORIAL
`-- unittest.vim



Any help is appreciated.

iz

**Comment by mattn** [comment]
are you setting timeoutlen option?

**Comment by izoj2013** [comment]
@mattn - thanks for your prompt feedback. Can you please point to the full documentation of emmet-vim; where does one configure the 'timeoutlen'? Are there other settings one needs to know?

iz

**Comment by mattn** [comment]
not yet in doc. if you want, please see issue list.

**Comment by samthom** [comment]
I have the same issue with emmet with vundle installation.
Got any solution ?

**Comment by mattn** [comment]
No. Modifying 'timeoutlen' effect to all of vim plugins which using key mappings. Not for only emmet.

**Comment by samthom** [comment]
No Never modified _timeoutlen_ infact other plugins like **nerdTree** and **ctrlP** is working fine.

**Comment by mattn** [comment]
Then, why you thought ` the same issue` ?

**Comment by samthom** [comment]
Third comment.
Same happened to me.
Installed and tried first tutorial.

**Comment by mattn** [comment]
AFAIK, most of cases that doesn't work emmet are:

* modifying timeoutlen
* installing another plugins and they break some conditions for emmet.

**Comment by samthom** [comment]
These are my plugins

**Plugin 'VundleVim/Vundle.vim'
Plugin 'vim-airline/vim-airline'
Plugin 'flazz/vim-colorschemes'
Plugin 'scrooloose/nerdtree'
Plugin 'kien/ctrlp.vim'
Plugin 'MarcWeber/vim-addon-mw-utils'
"Plugin 'tomtom/tlib_vim'
Plugin 'garbas/vim-snipmate'
"Plugin 'honza/vim-snippets'
Plugin 'tpope/vim-surround'
Plugin 'tmhedberg/matchit'
Plugin 'powerline/fonts'
Plugin 'vim-airline/vim-airline-themes'
Plugin 'mattn/emmet-vim'
Plugin 'pangloss/vim-javascript'
Plugin 'airblade/vim-gitgutter'
Plugin 'KabbAmine/zeavim.vim'**

**Comment by mattn** [comment]
Please try to check which plugin stop emmet.

**Comment by samthom** [comment]
Ok, Thank you for your response.
Hope I can fix this.

**Comment by mattn** [comment]
Sorry, I don't use most of your's.
